src/screens/auth/Splash.tsx

```diff
@@ -39,11 +35,11 @@ const Login: React.FC = () => {
   ];
 
   const handleRedirect = () => {
-    navigate("/signup");
+    navigate("/signin");
   };
 
   return (
-    <Layout isNavbar={false}>
+    <Layout isNavbar={false} isBottombar={false}>
```

Comment:
if nav and bottom bar is false then what is the purpose of layout use here?

Reply:
we have added style on navbar so that if we remove then splash screen layout will break so that we need to keep as it is.